Building DAG of jobs...
Provided cores: 1
Rules claiming more threads will be scaled down.
Unlimited resources: walltime, mem
Job counts:
	count	jobs
	2	align
	1	all
	3

rule align:
    input: data/adap_uBAM/Project_TBDY_L1_DTDB05_H1220P_York_1/GN2_S6_L001.adap.bam, refGenome/BwaIndex/genome.fa, refGenome/gatkIndex/genome.fa
    output: data/mapped_reads/Project_TBDY_L1_DTDB05_H1220P_York_1/GN2_S6_L001.bam
    jobid: 9
    wildcards: fragment=Project_TBDY_L1_DTDB05_H1220P_York_1/GN2_S6_L001
    resources: walltime=28800, mem=32000000000

Error in rule align:
    jobid: 9
    output: data/mapped_reads/Project_TBDY_L1_DTDB05_H1220P_York_1/GN2_S6_L001.bam

RuleException:
CalledProcessError in line 276 of /home/aparigi/dogCancer/snakemake_pipeline/Snakefile:
Command ' set -euo pipefail;  
        #module load Java/jdk1.8.0
        set +eu
	source activate gatk4.1
	set +eu
        #module load bwa/0.7.7.r441
        gatk --java-options "-Xmx30G" SamToFastq         -I=data/adap_uBAM/Project_TBDY_L1_DTDB05_H1220P_York_1/GN2_S6_L001.adap.bam         --FASTQ=/dev/stdout         --CLIPPING_ATTRIBUTE=XT --CLIPPING_ACTION=2 --INTERLEAVE=true -NON_PF=true         --TMP_DIR="tmp3/Project_TBDY_L1_DTDB05_H1220P_York_1/GN2_S6_L001" |         bwa mem -M -t 1 -p refGenome/BwaIndex/genome /dev/stdin |         gatk --java-options "-Xmx30G" MergeBamAlignment         --ALIGNED_BAM=/dev/stdin         --UNMAPPED_BAM=data/adap_uBAM/Project_TBDY_L1_DTDB05_H1220P_York_1/GN2_S6_L001.adap.bam         --OUTPUT=data/mapped_reads/Project_TBDY_L1_DTDB05_H1220P_York_1/GN2_S6_L001.bam         -R=refGenome/gatkIndex/genome.fa --CREATE_INDEX=true --ADD_MATE_CIGAR=true         --CLIP_ADAPTERS=false --CLIP_OVERLAPPING_READS=true         --INCLUDE_SECONDARY_ALIGNMENTS=true --MAX_INSERTIONS_OR_DELETIONS=-1         --PRIMARY_ALIGNMENT_STRATEGY=MostDistant --ATTRIBUTES_TO_RETAIN=XS         --TMP_DIR="tmp4/Project_TBDY_L1_DTDB05_H1220P_York_1/GN2_S6_L001" ' returned non-zero exit status 4.
  File "/home/aparigi/dogCancer/snakemake_pipeline/Snakefile", line 276, in __rule_align
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Will exit after finishing currently running jobs.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2021-02-09T083921.285264.snakemake.log
